src/bench/coin_selection.cpp

```diff
@@ -47,9 +47,12 @@ static void CoinSelection(benchmark::State& state)
         CAmount nValueRet;
         bool bnb_used;
         CoinEligibilityFilter filter_standard(1, 6, 0);
-        CoinSelectionParams coin_selection_params(false, 34, 148, CFeeRate(0), 0);
-        bool success = wallet.SelectCoinsMinConf(1003 * COIN, filter_standard, vCoins, setCoinsRet, nValueRet, coin_selection_params, bnb_used)
-                       || wallet.SelectCoinsMinConf(1003 * COIN, filter_standard, vCoins, setCoinsRet, nValueRet, coin_selection_params, bnb_used);
+        CoinSelectionParams coin_selection_params_pass_1(true, 34, 148, CFeeRate(0), 0);
+        bool success = wallet.SelectCoinsMinConf(1003 * COIN, filter_standard, vCoins, setCoinsRet, nValueRet, coin_selection_params_pass_1, bnb_used);
+        if (!success) {
+            CoinSelectionParams coin_selection_params_pass_2(false, 34, 148, CFeeRate(0), 0);
```

Comment:
Looks like this is a 2 pass algorithm 😛 

Why not just `coin_selection_params.use_bnb = false;` and keep the same params?